src/declarations/stencil-private.ts

```diff
@@ -1,7 +1,7 @@
 import { result } from '@utils';
-import type { Serializable as CPSerializable } from 'child_process';
 
 import type { InMemoryFileSystem } from '../compiler/sys/in-memory-fs';
+import type { CPSerializable } from './child_process';
```

Comment:
this is a bit convoluted, I know, but the alternative is to keep importing directly from `child_process` in this file and then run `dts-bundle-generator` on the whole thing (`build/declarations/stencil-private.d.ts`). In some rough testing this took the size of `internal/stencil-private.d.ts` from ~69k to ~160k, which I thought was too much. So instead we stand up a module that just re-exports one type from `child_process`, and we can then bundle _that_ with `dts-bundle-generator` and ship it right next to `stencil-private.d.ts` so it can import the inlined type from it, making `stencil-private.d.ts` portable.